services/apps/src/apps/routes/admin.clj

```diff
@@ -243,7 +259,7 @@
   "metadata"
   "POST /ontologies/{ontology-version}/{root-iri}/filter-unclassified"))
         (ok (coerce! AppListing
-                     (listings/get-unclassified-app-listing current-user ontology-version root-iri params)))))
+                     (listings/get-unclassified-app-listing current-user ontology-version root-iri attr params)))))
```

Comment:
The logic of the `get-unclassified-app-listing` function has not changed. Its params were only changed to be consistent with the new `list-apps-under-hierarchy` functions.